docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,197 @@
+---
+title: How to create a .NET Core Global Tool
+description: Describes how to create a Global Tool. The Global Tool is a console application that is installed through the .NET Core CLI.
+author: Thraka
+ms.author: adegeo
+ms.date: 08/22/2018
+---
+
+# Create a .NET Core Global Tool
+
+This article teaches you how to create and package a .NET Core Global Tool. The .NET Core CLI allows you to create a console application as a Global Tool, which others can easily install. .NET Core Global Tools are NuGet packages that are installed from the .NET Core CLI. For more information about Global Tools, see [.NET Core Global Tools overview][global-tool-info].
+
+[!INCLUDE [topic-appliesto-net-core-21plus.md](../../../includes/topic-appliesto-net-core-21plus.md)]
+
+## Create a project
+
+This article uses the .NET Core CLI to create and manage a project. Obviously you could use Visual Studio to do some of these steps, but Visual Studio will not be described in this article.
+
+Our example tool will be a console application that generates an ascii bot and prints a message. First, create a new .NET Core 2.1 Console Application.
+
+```console
+dotnet new console -o botsay
+```
+
+Enter the `botsay` directory created by the previous command.
+
+## Add the code
+
+Open the `Program.cs` file with your favorite text editor, such as `vim` or [Visual Studio Code](https://code.visualstudio.com/).
+
+Add the following `using` directive to the top of the file, this will help to display the version information of the application.
+
+```csharp
+using System.Reflection;
+```
+
+Next, move down to the `static void Main(string[] args)` method. Erase any code that exists inside this method, most likely you have a print statement in there from the console application template.
+
+Add the following code to process the command line arguments for your application. This code will first check if any arguments were passed. If no arguments were passed, a little help message will be displayed. If arguments were passed, all of those arguments will be transformed into a string and printed with the bot.
```

Comment:
remove future tense. This code first checks... perhaps move the explanation for after the code?

If arguments were passed -> Otherwise,

command line arguments -> command-line arguments